Uppsala/UppEt58.xml

```diff
@@ -0,0 +1,245 @@
+                                    <locus target="#1v"/>
+                                    <note>The written area only covers the upper half of the page.</note>
+                                </layout>
+                                
+                            </layoutDesc>
+                        </objectDesc>
+
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc><bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">146</citedRange>
+                                </bibl> calls it a "large or mid-sized very clear writing" (<foreign xml:lang="de">Große bis 
+                                mittelgroße sehr deutliche Schrift</foreign>). It is cautionsly dated to the 19th century by 
+                                    <bibl><ptr target="bm:alvin"/></bibl>.</desc>
+                                <date notBefore="1800" notAfter="1899">19th century.</date>
+                            </handNote>
+                        </handDesc>
+                        
+                        <decoDesc>
+                            
+                            <decoNote xml:id="d1" type="drawing">
+                                <locus target="1r"/>
+                                <desc>Seven seven-legged bows surround the text spiral on all sides.</desc>
+                            </decoNote>
+                            
+                            <decoNote xml:id="d2" type="drawing">
+                                <locus target="1v"/>
+                                <desc>Under the text on <locus target="#1v"/>, three crosses and one irregular symbol have been drawn.
+                                    The letters <foreign>ል</foreign>, <foreign>ቀ</foreign>, <foreign><sic resp="JK">ማ</sic></foreign>, 
+                                    and <foreign>ስ</foreign> appear around the crosses. The irregular symbol incorporates several 
+                                    instantiations of the letters <foreign>ተ</foreign> and <foreign>ሽ</foreign>.</desc>
+                            </decoNote>
+                            
+                            <decoNote xml:id="d3" type="drawing">
+                                <locus target="1v"/>
+                                <desc>Under <ref target="#d2"/>, a seven-times-seven grid filled with letters has been drawn. It
+                                appears to have been erased.</desc>
+                            </decoNote>
+                            
+                        </decoDesc>
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <desc type="Unclear">Note written with ink on the right side beneath the text spiral.</desc>
+                                    <q xml:lang="gez">
+                                        ይህን፡ ለመፍሔ፡ ጻፍ፡
```

Comment:
Isn't it a mixture of Gəʿəz and Amharic (ይህን፡, ጻፍ)? If ለመፍሔ stands for ለመፍትሔ, this could mean "write this for dissolving (the charm)"

Reply:
I also think it's better to mark it as Amharic